xpti/include/xpti_trace_framework.hpp

```diff
@@ -281,18 +240,18 @@ class PlatformHelper {
 
   bool checkTraceEnv() {
     bool enabled = false;
```

Comment:
Remove this variable and just explain in a Doxygen comment above what the function does.

Reply:
Done!